<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>File</string>
        </property>
        <widget class="QPushButton" name="openButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
        <widget class="QPushButton" name="saveButton">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Save Scatter</string>
         </property>
        </widget>
        <widget class="QPushButton" name="saveClustersButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Save Clusters</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>180</height>
         </size>
        </property>
        <property name="title">
         <string>DBScan Analysis</string>
        </property>
        <widget class="QPushButton" name="exportAllDistancesButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>121</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Export All Distances</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clusterButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>121</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Cluster</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Epsilon Distance</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Minimum Points Per Cluster</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Minimum Neighbor Points</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="minPointsSpin">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>80</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="minNeighborsSpin">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>110</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="epsilonSpin">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>50</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="exportNearestButton">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>20</y>
           <width>141</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Export Nearest Distances</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_Clusters"/>
    <addaction name="actionSave_Scatter"/>
   </widget>
   <widget class="QMenu" name="menuDBScan">
    <property name="title">
     <string>DBScan</string>
    </property>
    <addaction name="actionCluster"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDBScan"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionCluster">
   <property name="text">
    <string>Cluster</string>
   </property>
  </action>
  <action name="actionSave_Clusters">
   <property name="text">
    <string>Save Clusters</string>
   </property>
  </action>
  <action name="actionSave_Scatter">
   <property name="text">
    <string>Save Scatter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
